### PR TITLE
First Draft of Lotame Panorama ID Documentation

### PR DESCRIPTION
**Pull request by markaconrad**

### CODE REVIEW

File: dev-docs/modules/userId.md [hunk 1]
```diff
@@ -67,7 +68,7 @@ of sub-objects. The table below has the options that are common across ID system
 {: .table .table-bordered .table-striped }
 | Param under userSync.userIds[] | Scope | Type | Description | Example |
 | --- | --- | --- | --- | --- |
-| name | Required | String | May be: `"britepoolId"`, `"criteo"`, `"digitrust"`, `"id5id"`, `identityLink`, `"liveIntentId"`, `"parrableId"`, `"netId"`, `"pubCommonId"`,  or `"unifiedId"` | `"unifiedId"` |
+| name | Required | String | May be: `"britepoolId"`, `"criteo"`, `"digitrust"`, `"id5id"`, `identityLink`, `"liveIntentId"`,`"lotamePanoramaId"` `"parrableId"`, `"netId"`, `"pubCommonId"`,  or `"unifiedId"` | `"unifiedId"` |
```
Comment (ErikMHummel) [inline dev-docs/modules/userId.md]: Needs a comma.

File: dev-docs/modules/userId.md [hunk 2]
```diff
@@ -25,6 +25,7 @@ The User ID module supports multiple ways of establishing pseudonymous IDs for u
 * **ID5 Universal ID** - a neutral identifier for digital advertising that can be used by publishers, brands and ad tech platforms (SSPs, DSPs, DMPs, Data Providers, etc.) to eliminate the need for cookie matching.
 * **Identity Link** – provided by LiveRamp, this module calls out to the ATS (Authenticated Traffic Solution) library or a URL to obtain the user’s IdentityLink envelope.
 * **LiveIntent ID** – a user identifier tied to an active, encrypted email in our graph and functions in cookie-challenged environments and browsers.
+* **Lotame Panorama ID** - a user ID that does not depend on 3rd party cookies provided by Lotame.
```
Comment (ErikMHummel) [inline dev-docs/modules/userId.md]: Do we want to mention it is a unique individual across devices? Not sure how much marketing speak to put here. But looking at others, they seems to use it pretty heavily.

File: dev-docs/modules/userId.md [hunk 3]
```diff
@@ -494,6 +495,36 @@ pbjs.setConfig({
 })
 ```
 
+### Lotame Panorama ID
+
+The Lotame Panorama ID module enables publishers to exchange data with the ad tech ecosystem without depending on 3rd party cookies.
```
Comment (ErikMHummel) [inline dev-docs/modules/userId.md]: Do we want to link to https://www.lotame.com/panorama/ or https://www.lotame.com? 

Maybe a follow-on sentence of:

> For more information, please see the [Lotame Panorama](https://www.lotame.com/panorama/) product site.

**Review by ErikMHummel — CHANGES_REQUESTED** [review]
We should probably get some marketing speak from Charlie and then we can tone it down a little to match the other company's sections.

File: dev-docs/modules/userId.md (hunk 1 above)
Comment (markaconrad) [reply dev-docs/modules/userId.md]: Fixed

File: dev-docs/modules/userId.md (hunk 3 above)
Comment (markaconrad) [reply dev-docs/modules/userId.md]: Updated with wording from Charlie

File: dev-docs/modules/userId.md (hunk 2 above)
Comment (markaconrad) [reply dev-docs/modules/userId.md]: Updated with wording from Charlie

**Review by ErikMHummel — APPROVED** [review]
I'm good now Mark.

**Pull request closed without merging**